src/fontra/views/editor/panel-transformation.js

```diff
@@ -451,6 +530,77 @@ export default class TransformationPanel extends Panel {
     };
   }
 
+  async doPathOperations(pathOperationFunc, undoLabel) {
+    let { point: pointIndices } = parseSelection(this.sceneController.selection);
+    pointIndices = pointIndices || [];
+
+    const positionedGlyph =
+      this.sceneController.sceneModel.getSelectedPositionedGlyph();
+
+    const allContourIndices = [...range(positionedGlyph.glyph.path.numContours)];
+    const selectedContourIndices =
+      !pointIndices.length && undoLabel === "Remove overlaps"
+        ? allContourIndices
+        : [...getSelectionByContour(positionedGlyph.glyph.path, pointIndices).keys()];
+
+    if (!selectedContourIndices.length) {
+      return;
+    }
+
+    const notSelectedContourIndices = allContourIndices.filter(
+      (n) => !selectedContourIndices.includes(n)
+    );
+    if (!notSelectedContourIndices.length && undoLabel !== "Remove overlaps") {
+      return;
+    }
+
+    const editLayerGlyphs = this.sceneController.getEditingLayerFromGlyphLayers(
+      positionedGlyph.varGlyph.glyph.layers
+    );
+
+    const layerPaths = await mapObjectValuesAsync(
+      editLayerGlyphs,
+      async (layerGlyph) => {
+        const path = layerGlyph.path;
+        const selectedContoursPath = new VarPackedPath();
+        const unselectedContoursPath = new VarPackedPath();
+
+        for (const contourIndex of range(path.numContours)) {
+          if (selectedContourIndices.includes(contourIndex)) {
+            selectedContoursPath.appendContour(path.getContour(contourIndex));
+          } else {
+            unselectedContoursPath.appendContour(path.getContour(contourIndex));
+          }
+        }
+
+        return await pathOperationFunc(selectedContoursPath, unselectedContoursPath);
```

Comment:
`unionPath()` only takes one argument, so this looks misleading.

Reply:
I would also love you to reverse the arguments: subtractPaths is now backwards (hence you use REVERSE_DIFFERENCE in the implementation). subtractPaths(pathA, pathB) should subtract pathB from pathA.

Reply:
ok, done.